test/config-next/ra.json

```diff
@@ -127,7 +126,6 @@
 		},
 		"features": {
 			"AsyncFinalize": true,
-			"UseKvLimitsForNewOrder": true,
```

Comment:
Make sure this flag _is_ set in the corresponding //test/config/ra.json, to reflect the fact that it's currently set in prod. Same comment for SA and WFE below.